docs/FAQ.md

```diff
@@ -137,6 +137,17 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I resume working on an existing change?
+
+There are two ways to resume working on an earlier change: `jj new` then `jj amend`,
+and `jj edit`. The first is generally recommended over the second for a few reasons,
+which come down to how intentional you are with your changes. If the revision you 
+want to resume working on has a conflict, `jj edit` would restore the state with
+plain-text conflict annotations. Editing those annotations incorrectly or by
+accident can lead to undesired output from conflict resolution. If you amend a new
+revision onto a revision that contains a conflict, the conflict will be updated to
+correctly reflect your new changes.
```

Comment:
I think "amend" works better with "into" than "onto" (I'm confused about what this sentence means, but I can guess from context).

How about the following?

> Instead, create a new commit on top of the conflicted commit. Then resolve the conflicts there (and run tests etc.) and squash the resolution into the conflicted commit when you're done.

Does that seem readable to you?

Reply:
I'm realizing that I'm using a very different prose than the rest of the FAQ. I've been drafting using a "we're discussing theory" prose, as opposed to a prescriptive prose. I wouldn't use a prescriptive prose for a context where we want to communicate deeper understanding, but I think that that is likely a discussion for another context. I'm going to take another crack at this using a prescriptive prose, see if that makes it easier for us to land this PR.

Reply:
Updated, lmk what you think!